mock/src/main/java/com/tngtech/keycloakmock/impl/handler/LoginRoute.java

```diff
@@ -81,7 +85,16 @@ public void handle(@Nonnull final RoutingContext routingContext) {
       routingContext.put(
           AUTHENTICATION_URI,
           requestConfiguration.getAuthenticationCallbackEndpoint(session.getSessionId()));
-      renderHelper.renderTemplate(routingContext, "loginPage.ftl", "text/html");
+      engine
+          .render(routingContext.data(), "loginPage.ftl")
+          .onSuccess(
+              b ->
```

Comment:
strange formatting...

Reply:
Indeed, but the google formatter is quite adamant about this ;-)